app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -603,14 +603,14 @@ def repos_pending_sync(environment = nil, content_view = nil)
       end
 
       def rhsm_url
-        # Since Foreman 3.1 this setting is set
-        if (rhsm_url = setting(SmartProxy::PULP3_FEATURE, 'rhsm_url'))
-          URI(rhsm_url)
-          # Compatibility fall back
-        elsif pulp_primary?
-          URI("https://#{URI.parse(url).host}/rhsm")
-        elsif pulp_mirror?
-          URI("https://#{URI.parse(url).host}:8443/rhsm")
+        # Since Foreman 3.1 this setting is set. No value means direct use
```

Comment:
Open question: does this comment still add any value?

Reply:
IMHO not